definitions/npm/react-test-renderer_v16.x.x/flow_v0.47.x-/react-test-renderer_v16.x.x.js

```diff
@@ -44,6 +44,10 @@ type TestRendererOptions = {
   createNodeMock(element: React$Element<any>): any
 };
 
+type Thenable = {
+  then(resolve: () => mixed, reject?: () => mixed): mixed,
+};
+
```

Comment:
Please declare it inside the module declaration. We try to avoid global types now on. Especially with common names.